docs/sources/datasources/loki/query-editor/index.md

```diff
@@ -68,7 +68,7 @@ You can use the Loki label browser to navigate through your labels and values, a
 1. Choose a query type between [**logs query**]({{< relref "#create-a-log-query" >}}) and [**rate metrics query**]({{< relref "#create-a-metric-query" >}}).
    You can also validate the selector.
 
-{{< figure src="/static/img/docs/v75/loki_log_browser.png" class="docs-image--no-shadow" max-width="800px" caption="The Loki label browser" >}}
+{{< figure src="/static/img/docs/v75/loki_label_browser.png" class="docs-image--no-shadow" max-width="800px" caption="The Loki label browser" >}}
```

Comment:
Hahaha I was about to write you that this won't work because this is the name of file in grafana/website, but you already added a new screenshot there https://github.com/grafana/website/blob/master/static/static/img/docs/v75/loki_label_browser.png. 🚀 🚀 🚀 🚀 

Btw for the future, we usually add screenshots based on the version of Grafana. So `v75` means that it was added in version `7.5`. So for this screenshot, you could create `v93` folder and add it there. Just for the future. 🙂